src/modules/Categories/CategoryList.tsx

```diff
@@ -83,6 +82,10 @@ class CategoryList extends React.Component<Props, State> {
   someMethod() {
 
   }
+
+  searchCategory(filter:string) {
```

Comment:
searchCategory = (filter: string) => {
    this.props.actions.getCategories(filter);
}